src/components/app-bar/app-bar.atoms.tsx

```diff
@@ -0,0 +1,19 @@
+import type { HTMLAttributes } from 'react'
+import {
+  ElNavSearchButton,
+  ElNavSearchButtonIcon,
+  ElNavSearchButtonPlaceholder,
+  ElNavSearchButtonShortcutText,
+} from './styles'
+
+export interface NavSearchButton extends HTMLAttributes<HTMLButtonElement> {}
+
+export const NavSearchButton: React.FC<NavSearchButton> = (props) => {
+  return (
+    <ElNavSearchButton {...props}>
+      <ElNavSearchButtonIcon icon="search" />
```

Comment:
**question:** What is the `icon="search"` prop is doing here?

Reply:
sorry, I forgot to delete it after I tried the existing one